python-crfsuite 0.9.11, v1 recipe, add maintainer

bollwyvl opened the pull request:
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

References:
- noted on https://github.com/conda-forge/stanza-feedstock/pull/29
- includes #36
- includes #34

Changes:
- [x] `conda-forge.yml`
  - [x] v1 recipe (use `rattler-build`) 
  - [x] use `pixi`
  - [x] use `shellcheck`
- [x] `recipe`
  - [x] update to `0.9.11`
  - [x] use explicit script names
  - [x] include licenses for vendored libs
  - [x] run tests with coverage

regro-cf-autotick-bot committed:
updated v0.9.11

regro-cf-autotick-bot committed:
MNT: Re-rendered with conda-build 24.9.0, conda-smithy 3.40.1, and conda-forge-pinning 2024.10.02.13.37.37

regro-cf-autotick-bot committed:
Rebuild for python 3.14

regro-cf-autotick-bot committed:
MNT: Re-rendered with conda-smithy 3.52.0 and conda-forge-pinning 2025.08.28.20.53.1

bollwyvl committed:
pycrfsuite 0.9.11, v1 recipe

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.05.00.13.54

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
more v1 recipe

bollwyvl committed:
use shellcheck

bollwyvl committed:
merge #34

bollwyvl commented:
@conda-forge-admin please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18261705490](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18261705490). Examine the logs at this URL for more detail.</sub>

bollwyvl committed:
bump down coverage for py<3.14

bollwyvl commented:
@conda-forge/python-crfsuite ready for review

bollwyvl committed:
add self as maintainer

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.10.13.43.27

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.5

bollwyvl commented:
@conda-forge/core this PR (which should clear up all open PRs, some of which several years old) has been open for a week, and has received no review. Please consider a review, at your leisure.

hmaarrfk commented on recipe/recipe.yaml:
```diff
@@ -0,0 +1,68 @@
+    - if: build_platform != target_platform
+      then:
+        - python
+        - cross-python_${{ target_platform }}
```
```suggestion
        - cross-python_${{ target_platform }}
        - cython
```

don't you need cython as well here?

bollwyvl replied:
let's try!

bollwyvl replied:
(and also, thank you for the speedy and helpful review... you know me, just shaving yaks)

bollwyvl committed:
add cython to cross-compiling

Co-authored-by: Mark Harfouche <mark.harfouche@gmail.com>

bollwyvl commented:
@conda-forge-admin please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18432319229](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18432319229). Examine the logs at this URL for more detail.</sub>

bollwyvl commented:
Thanks!
